src/scripts/setup-iasql-db.sh

```diff
@@ -37,7 +37,9 @@ psql postgres://postgres:test@localhost:5432/iasql -c "
 echo "\nInstalling modules in iasql db..."
 psql postgres://postgres:test@localhost:5432/iasql -c "
   SELECT iasql_install(
-    'aws_ecs_simplified',
-    'aws_codebuild'
+    'aws_ecs_simplified',    
+    'aws_codebuild',
+    'aws_s3',
+    'aws_codepipeline'
```

Comment:
I wonder if we should have codebuild depend on code pipeline. For GH sources it is a no-op and doesn't hurt, but when the source is codepipeline does use it we can get some type safety

Reply:
project per se is not using codepipeline, see my next comment...